describe the process of starting the network (#474)
lukso-network/docs: docs/networks/mainnet.md

LUKSO's Blockchain Architecture consists of three clients:

- The beacon chain client running Casper the Friendly Finality Gadget (Casper FFG) as the consensus protocol (`Casper Consensus`: <https://github.com/lukso-network/prysm>)
- The execution engine client (EVM) (`Smart Contract Execution`: <https://github.com/lukso-network/go-ethereum>)
- The validator client (`Validator`)

## Becoming a Genesis Validator

LUKSO will start as a PoS Blockchain with an Initial Genesis Validator Set that anyone can join.

Once deployed on Ethereum, the Genesis Validator Smart contract will allow users to deposit LYXe and become Genesis Validators for LUKSO Mainnet. For each validator, you run, you must generate a validator key and deposit 32 LYXe. There will be no limit on the number of validators you can run.

> The window for becoming a Genesis Validator will be around 3 weeks before launch (or longer if needed). At least **4,096 genesis validators** are needed before the Genesis Validator Deposit Smart Contract can be frozen, and the process of genesis can begin.

Reviewer: ```suggestion
The window for becoming a Genesis Validator will be around 3 weeks before launch (or longer if needed). At least **4,096 genesis validators** are required before the Genesis Validator Deposit Smart Contract can be frozen and the process of genesis can begin.
```

> Once frozen, participants will be able to generate two files: genesis.ssz and genesis.json. These files will be the network’s starting point and determine the initial network state.

Reviewer: ```suggestion
Once frozen, participants can generate two files: genesis.ssz and genesis.json. These files will be the network’s starting point and determine the initial network state.
```

> A deposit dApp that reads from the Genesis Validator Deposit Smart Contract will be available for depositors. The initial network state will contain all Genesis Validator keys and be made available in the form of the genesis.ssz file.

Reviewer: ```suggestion
A deposit dApp that reads from the Genesis Validator Deposit Smart Contract will be available for depositors. The initial network state will contain all Genesis Validator keys and be available in the form of the genesis.ssz file, which can be downloaded after the deposit timeframe ended.
```

The second file, genesis.json, contains the initial state of the chain. This includes the initial balances and total supply of LYX. The community will have the choice of setting the initial supply at 35M, 42M (the founding team's suggestion), or 100M.

> The network start time will be defined in the genesis files. It will most likely be around 1 week after the Genesis Validator Deposit freeze. Validators must have their modes ready at least one day before the network start time. Nodes will automatically begin to create and receive blocks after the network start time.

Reviewer: ```suggestion
The network start time will be defined in the genesis files. It will likely be around 1 week after the Genesis Validator Deposit freeze. Validators must have their nodes ready at least one day before the network starts. Nodes will then automatically create and receive blocks after the network start time.
```
